tests/transformer/test_yield.py

```diff
@@ -0,0 +1,30 @@
+from RestrictedPython._compat import IS_PY3
+from tests import c_exec
+
+import pytest
+
+
+# Modified Example from http://stackabuse.com/python-async-await-tutorial/
+YIELD_FORM_EXAMPLE = """
+import asyncio
+
+@asyncio.coroutine
+def get_json(client, url):
+    file_content = yield from load_file('data.ini')
+"""
+
+
+@pytest.mark.skipif(
+    not IS_PY3,
+    reason="yield from statement was first introduced in Python 3.3")
+@pytest.mark.parametrize(*c_exec)
+def test_yield_from(c_exec):
```

Comment:
Besides `yield from` the test `test_transformer__RestrictingNodeTransformer__visit_Yield__1` in `test_transformer.py` for the `yield`  statemement should be moved here, too.